_episodes/01-shell-basics.md

```diff
@@ -8,42 +8,43 @@ questions:
 objectives:
 - "Understand basic differences among available shells and their use in
  neuroimaging toolkits"
-- "Using shell as a working medium explain the role of some most important environment variables"
+- "Using a shell as a working medium"
+- "Explain the role of some of the most important environment variables"
 - "Provide hints on efficient use of the collected shell history of commands"
 - "Explain how to make shell scripts more robust and less dangerous"
-- "Introduce basics of runtime- and unit- testing"
+- "Introduce basics of runtime and unit testing"
 keypoints:
-- "There are different (incompatible) shells and different
-  neuroimaging tools might use a different shell and thus provide
-  instructions which would not be compatible with your current shell"
-- "Command line shell is a powerful tool, and learning additional
+- "There are a number of incompatible shells. Different
+  neuroimaging tools may use specific shells and thus provide
+  instructions that are not compatible with your current shell."
+- "A command line shell is a powerful tool and learning additional
  'tricks' could help to make its use much more efficient, less
- error-prone, and thus more reproducible"
+ error-prone, and thus more reproducible."
```

Comment:
yeah -- saw semicolons ;-)  I will let them be although I would have just kept periods interim sentences.
you might then like also following aliases. 
```
    wdiff = diff --color-words
    pdiff = diff 'HEAD@{1}'
    pdfdiff = difftool -x diffpdf --no-prompt
    niidiff = difftool -x niidiff --no-prompt
    kompare = difftool -x kompare --no-prompt
    nbdiff = difftool -x nbdiff --no-prompt
```
I do use kompare also from time to time if I want quick GUI view and there is not that many files changed